apis/bitbucketapi/apis.go

```diff
@@ -0,0 +1,83 @@
+package bitbucketapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/kubescape/go-git-url/apis"
+)
+
+const (
+	DEFAULT_HOST string = "bitbucket.org"
+)
+
+type IBitBucketAPI interface {
+	GetRepoTree(owner, repo, branch string, headers *Headers) (*Tree, error)
+	GetDefaultBranchName(owner, repo string, headers *Headers) (string, error)
+	GetLatestCommit(owner, repo, branch string, headers *Headers) (*Commit, error)
+}
+
+type BitBucketAPI struct {
+	httpClient *http.Client
+}
+
+func NewBitBucketAPI() *BitBucketAPI { return &BitBucketAPI{httpClient: &http.Client{}} }
+
+func (gl *BitBucketAPI) GetRepoTree(owner, repo, branch string, headers *Headers) (*Tree, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
```suggestion
        return nil, fmt.Errorf("GetRepoTree is not supported")
```
This function should return an error indicating that it is not implemented, but not panic.